allennlp/modules/text_field_embedders/basic_text_field_embedder.py

```diff
@@ -40,7 +40,9 @@ def forward(self, text_field_input: Dict[str, torch.Tensor]) -> torch.Tensor:
                                                             str(text_field_input.keys()))
             raise ConfigurationError(message)
         embedded_representations = []
-        for key, tensor in text_field_input.items():
+        keys = sorted(text_field_input.keys())
```

Comment:
why not just `for key, tensor in sorted(text_field_input.items())` ?

Reply:
Yes, that's probably also good, mine was a variant from earlier when we wanted to dictate a particular order (tokens before token_characters to match a different model). Maybe this is slightly clearer, that we want to order the keys in a particular order.

Reply:
I'm cool either way